pages/design/Design Guidelines - Designers.md

```diff
@@ -0,0 +1,221 @@
+
+#### [Lorem ipsum – Figma](https://www.figma.com/community/plugin/736000994034548392/Lorem-ipsum)
+Lorem ipsum is a plugin that lets you insert dummy text. We use this plugin where we want to put the text before the client sends the final version of the text. 
+
+#### [Unsplash – Figma](https://www.figma.com/community/plugin/738454987945972471/Unsplash)
+Unsplash is a platform that has commercial free to use images. You can search for images in the plugin and it will be imported as an high quality image.
+
+#### [Select Layers](https://www.figma.com/community/plugin/799648692768237063)
+Select Layers is a very helpful plugin that can offer Layer selection based on a lot of criteria. The most that we use is "Select Layers by Name", "Select Text Layers", "Select Locked Layers", "Select Instances" and "Select Hidden Layers".
+
+# Project Starter
+Almost all of our projects are made with our Project Starter. Our Project Starter is a Figma project made specifically by us, and adapted for the technologies we use to develop.
+
+Before you start to design, make sure to duplicate the Project Starter to your drafts or Team.
+
+In our Starter, you will find:
+- Thumbnail page
+- Components and Variants
+- Color Styles
+- Text Styles
+- Buttons
+- Input Groups
+- Alerts
+- Paginaters
+- Navigation Bars
+- Footers
+- Modals
+- [V3 TailwindCSS Color Palette](https://tailwindcss.com/docs/customizing-colors)
+- [V3 TailwindCSS Shadows](https://tailwindcss.com/docs/drop-shadow)
+- 404 Page
+- OG/Meta Image
+- Favicon
+- [12/4 Column Grid system · Bootstrap 4)](https://getbootstrap.com/docs/4.0/layout/grid/)
+- Cookie Notice Modal
+
+## Thumbnail
+On the Thumbnail Page, you will find:
+- Layout Grid
+- Company logo
+- Status Structure
+- Status Types
+- Project Name
+
+#### Layout Grid
+As a Layout Grid, we have inputed the Safe Zone where you can put the content in it, so you make sure the content doesn't cut out.
+
+#### Company Logo
+Company logo is presented via a 64x64 Frame, that you could change in order to display to the thumbnail.
+
+#### Status Structure
+The status structure component is made as a structure for the Status Types. Adjusting its properties, all the Status Type Components will change based on the Status Structure Component.
+
+#### Status Types
+We use status types in order to indicate via the thumbnail the status of the project. The number of the Status Types is limitless, feel free to add and remove how many status types you need. Just make sure to stick to `Status / <yourname>` rename structure.
+
+#### Project Name
+The project name is the title of the project that you want to display on the thumbnail.
+
+#### Last Step
+After you made all your changes, make sure to select **Cover / Master** Frame, right clicking on it and then selecting **Set as Thumbnail**. The changes will apply after you restart Figma app.
+
+
+## Components and Variants
```

Comment:
Most of these are not variants they can be called partials

Reply:
there is not a single word named "partial" in Figma